.travis.yml

```diff
@@ -14,6 +14,15 @@ branches:
 
 matrix:
   include:
+    - php: 5.3
+      dist: precise
+      env: DEPENDENCIES='low'
+    - php: 5.4
+      dist: trusty
+      env: DEPENDENCIES='low'
```

Comment:
I suggest removing these `DEPENDENCIES='low'`. We should rather test on higher (compatible) versions than on lowest ones, to match what our users will get (we need one job using `DEPENDENCIES='low'` in our matrix, but if you look at it, you will see that it also has other normal jobs running on that PHP version)

Reply:
Done